study/hello-ppx_rapper/README.md

```diff
@@ -0,0 +1,56 @@
+# Hello, ppx_rapper!
+
+Let's take a look at [ppx_rapper](https://github.com/roddyyaga/ppx_rapper)!.
+
+We have already seen what PPXs are, so we are good to go!
+
+Have a look at the `Exec` module. We converted queries from the [hello-caqti](hello-caqti-lwt) project, to a much shorter syntax. Although, we have to adjust the SQL slightly in order to satisfy ppx_rapper's requirements.
+
+Also take a look `Fake_users` module for other types of usage, and as always the tests.
```

Comment:
I would consider translating a more realistic example to avoid the syntactic limitation, since I don't think its something which turns up in real usage.  However, I suggest to spending too much time on this at the moment since it may be easier to pick a good case to translate when the main narrative of the tutorial is in place.  That is, feel free to commit as-is for now.

Reply:
Sounds good. So I'm merging for now and we can come up with better examples a bit later.